pkg/solana/txm/pendingtx.go

```diff
@@ -211,12 +206,32 @@ func (c *pendingTxContext) Remove(sig solana.Signature) (id string, err error) {
 	})
 }
 
-func (c *pendingTxContext) ListAll() []solana.Signature {
+func (c *pendingTxContext) ListAllSigs() []solana.Signature {
 	c.lock.RLock()
 	defer c.lock.RUnlock()
 	return maps.Keys(c.sigToID)
 }
 
+func (c *pendingTxContext) ListAllTxsIDs() []string {
```

Comment:
Noticed this is only really used in a test. Could we just use `maps.Values(c.sigToID)` directly to avoid adding a new method? No worries if there's a risk of race or anything.

Reply:
removed!